users/details/script.js

```diff
@@ -350,6 +375,28 @@ function generateReadableDateFromSecondsTimeStamp(timeStamp) {
   //created function for readable date format
   return new Date(timeStamp * 1000).toDateString(); // new function because we are getting the value in seconds and not milliseconds
 }
+console.log(generateReadableDateFromSecondsTimeStamp('1688445662'));
```

Comment:
Please explain why you put console log here?

Reply:
Sorry about this. Changed.